packages/back-end/src/controllers/stripe.ts

```diff
@@ -161,8 +197,15 @@ export async function postWebhook(req: Request, res: Response) {
     case "checkout.session.completed": {
       const { subscription } = event.data
         .object as Stripe.Response<Stripe.Checkout.Session>;
+
       if (subscription) {
         updateSubscription(subscription);
+      } else {
+        console.error("Unable to find & updated existing organization"); //TODO: As this is a webhook, these errors should probably alert/bubble up somewhere
+        res.status(400).json({
+          status: 400,
+          message: "Unable to find & updated existing organization",
+        });
```

Comment:
I don't think this `else` block does what you think it does.  This isn't checking to see if the database has a copy of the subscription already.  It's just a sanity check to make sure the stripe event contains a subscription field, which it should always contain.

Reply:
Updated. When I updated, I saw that the logic for this case was the same as the existing invoice.paid and invoice.payment_failed cases, so I just added checkout.session.completed to the previously mentioned cases.